Sanity-Vercel/docs/nextjs/deploy.md

```diff
@@ -1,4 +1,10 @@
+### Deploying through Vercel
+
+Create an account in Vercel (https://vercel.com/), it's free!
+
 Your project will be hosted by vercel and we will use the vercel CLI to deploy. 
 
 Deploying your solution using the vercel CLI
```

Comment:
I find this sentence a bit unclear, when it's the -ing form it feels like a title? We should either highlight it in that case or make into a step by saying "Deploy"?

Reply:
removed the -ing